apps/settings/elements/bluetooth_v2.html

```diff
@@ -8,20 +8,18 @@ <h1 data-l10n-id="bluetooth-header"></h1>
     <div>
       <ul>
         <li class="bluetooth-status hint">
-          <label class="pack-switch">
-            <input type="checkbox" data-ignore>
-            <span data-l10n-id="bluetooth">Bluetooth</span>
-          </label>
+          <gaia-switch data-ignore>
+            <label data-l10n-id="bluetooth">Bluetooth</label>
```

Comment:
Since data-l10n-id will replace the content, we could remove the `Bluetooth` text